src/averageAges.js

```diff
@@ -37,7 +43,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const mother = people.map(p => p.mother).filter(p => p !== null);
```

Comment:
it could not be mother
mother is a singular
mothers is a plural

people.map() return array, so name it `mothers`

what is p? 
```suggestion
  const mothers = people
    .map(person => person.mother)
    .filter(person => person !== null);
```